docs/routing.md

```diff
@@ -39,7 +39,7 @@ Route('/users/{username}', user)
 ```
 By default this will capture characters up to the end of the path or the next `/`.
 
-You can use convertors to modify what is captured. Four convertors are available:
+You can use convertors to modify what is captured. Five convertors are available:
```

Comment:
Maybe remove that number completely so we won't face this again?
I mean the list is displayed right here, what's the importance of counting it?

Reply:
Yes, That's would be much better.
Do you wanna me do it ?

Reply:
If you're willing, it would be cool. 😁

Reply:
Done.